components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -377,11 +385,12 @@ protected boolean handlePostAuthentication(Event event, String userName, UserSto
     /**
      * Check whether the account can be unlocked by checking whether account unlock time is exceeded or account bypass
      * role is attached
-     * @param userName name of the logged in user
-     * @param userStoreManager user store
+     *
+     * @param userName              name of the logged in user
```

Comment:
Shall we add a `.` at the end of the line and start the sentence with an uppercase letter?
Shall we fix other places as well?